packages/calypso-e2e/src/data-helper.ts

```diff
@@ -37,6 +37,16 @@ export function getCalypsoURL(
 	return url.toString();
 }
 
+/**
+ * Wrapper around the `config.get` call.
+ *
+ * @param {string} key Top level key in the decrypted configuration file to get.
+ * @returns {[key: string]: any|string|null} Either a JSON dictionary or string matching the key if the key exists. Returns null otherwise.
+ */
+export function getConfig( key: string ): { [ key: string ]: any } | string | null {
+	return config.get( key );
```

Comment:
This wrapper is simply so the calling methods do not have to import `config` themselves.
It is assumed that the calling method checks the returned value.

Reply:
What if, instead of creating another abstraction layer, we directly export the `config` object?

```js
export { config };
```

Reply:
Ah, didn't know that was  an option!

Reply:
Addressed!